README.md

```diff
@@ -20,6 +20,8 @@ This workspace has been generated by [Nx](https://nx.dev) so you can run the fol
 
 You can also install [Nx Console](https://marketplace.visualstudio.com/items?itemName=nrwl.angular-console&utm_source=nx-project) in VSCode to easily access Nx commands directly in your code editor.
 
-For specific documentation for each app read the following documents:
+Probably more configuration/setup is needed before running the app. For specific documentation for each app read the following documents:
```

Comment:
Maybe probably is not the best word since this read-me only has the setup for the workspace and not the individual apps, the developer will 100% need to check the documentation for the specific app in order to run it